src/main/java/billgates/entities/Entry.java

```diff
@@ -0,0 +1,14 @@
+package billgates.entities;
+
+import java.time.ZonedDateTime;
+
+public class Entry extends AbstractEntry {
+
+    protected Entry(Attribute<Integer> id, Attribute<ZonedDateTime> date, Attribute<Double> value,
+                 Attribute<String> currency, Attribute<String> description,
+                 Attribute<String> from, Attribute<String> to, Attribute<String> location) {
+        super(id, date, value, currency, description, from, to, location);
```

Comment:
Remember that this is a TODO

Reply:
That there is a splitter to be implemented under the entry class
Link is here: 
https://github.com/CSC207-2022F-UofT/course-project-bill-gates/pull/15/files#diff-9fddc4b7d153a3e58f867ac775ad71a92c63d3c9a6f9c08aa25d242db54e6a2fR10